src/components/Login/ForgotPasswordSubmit/index.js

```diff
@@ -1,58 +1,61 @@
-import React, { Component } from "react";
-import { Auth } from "aws-amplify";
+import React, { useState } from "react";
+import { connect } from "react-redux";
 
-import Session from "../../../utility/constants/Session";
 import Routes from "../../../utility/constants/Routes";
+import { userActions } from "../../../Redux/actionCreators";
+import ErrorMsgBox from "../../common/ErrorMsgBox";
 
-class ForgotPasswordSubmit extends Component {
-  state = {
-    code: "",
-    password: "",
-  };
+const ForgotPasswordSubmit = ({ history, error, username, forgotPasswordSubmit }) => {
+  const [code, setCode] = useState();
+  const [password, setPassword] = useState();
 
-  handleCode = event => {
-    this.setState({ code: event.currentTarget.value });
+  const handleCode = event => {
+    setCode(event.currentTarget.value);
   };
 
-  handlePassword = event => {
-    this.setState({ password: event.currentTarget.value });
+  const handlePassword = event => {
+    setPassword(event.currentTarget.value);
   };
 
-  handleSubmit = event => {
-    event.preventDefault();
-    let username;
-    if (sessionStorage.getItem(Session.USERNAME)) {
-      username = sessionStorage.getItem(Session.USERNAME);
-    }
-    const { code, password } = this.state;
+  const handleSubmit = event => {
     event.preventDefault();
-    Auth.forgotPasswordSubmit(username, code, password).then(res => {
-      this.props.history.push(Routes.AI_MARKETPLACE);
-    });
+    const route = `/${Routes.AI_MARKETPLACE}`;
+    forgotPasswordSubmit({ username, code, password, history, error, route });
   };
 
-  render() {
-    const { code, password } = this.state;
-    return (
-      <div>
-        <form>
-          <div>
-            <label htmlFor="code">Enter Code to Reset Password</label>
-            <input id="code" type="password" value={code} onChange={this.handleCode} />
-          </div>
-          <div>
-            <label htmlFor="password">Enter the New Password</label>
-            <input id="password" type="password" value={password} onChange={this.handlePassword} />
-          </div>
-          <div>
-            <button type="submit" onClick={this.handleSubmit}>
-              Validate
-            </button>
-          </div>
-        </form>
-      </div>
-    );
-  }
-}
-
-export default ForgotPasswordSubmit;
+  return (
+    <div>
+      <form>
+        <div>
+          <label htmlFor="code">Enter Code to Reset Password</label>
+          <input id="code" type="password" value={code} onChange={handleCode} />
+        </div>
+        <div>
+          <label htmlFor="password">Enter the New Password</label>
+          <input id="password" type="password" value={password} onChange={handlePassword} />
+        </div>
+        <div>
+          <button type="submit" onClick={handleSubmit}>
+            Validate
+          </button>
+        </div>
+        <p>fghjk{error}</p>
```

Comment:
Potential typo?

Reply:
removed it in the forgot-password [PR](https://github.com/Vivek205/refactored-dapp/pull/28)